rollup/sync_service.go

```diff
@@ -951,20 +1007,47 @@ func (s *SyncService) ProcessSequencerBatchAppendedLog(ctx context.Context, ethl
 					return fmt.Errorf("Unexpected type when parsing ctc tx eip155: %T", ctcTx.tx)
 				}
 				nonce, gasLimit := uint64(eip155.nonce), uint64(eip155.gasLimit)
-				to, l1TxOrigin := eip155.target, common.Address{}
+				to := eip155.target
```

Comment:
Make sure sequencer rejects transactions sent directly to ZERO_ADDRESS. Nothing good comes from sending to the zero address!!!!

Reply:
The sequencer will not accept txs sent to the zero address from queue origin sequencer as of c508935484d078fb9fbf767cdbc2239646f20bd8

This needs to apply to queue origin l1 to l2 as well? We would need to revert the tx instantly in the ovm I think to solve this problem